kinto/tests/test_views_records.py

```diff
@@ -203,3 +203,27 @@ def test_record_is_accessible_by_group_member(self):
         self.app.get(self.record_url,
                      headers=self.aaron_headers,
                      status=200)
+
+    def test_records_should_reject_unaccepted_request_content_type(self):
+        headers = self.headers.copy()
+        headers['Content-Type'] = 'text/plain'
+        self.app.put(self.record_url,
+                     MINIMALIST_RECORD,
+                     headers=headers,
+                     status=415)
+
+    def test_records_should_reject_unaccepted_client_accept(self):
```

Comment:
I can also duplicate this code in groups/buckets and collection if you think it is reasonable to do it.

Since the code is in Cliquet it is unlikely that they will fail if the other one passes.
It is just some functional tests to make sure that what we want for Kinto is working regardless of cornice or cliquet changes.